pypeit/sensfunc.py

```diff
@@ -264,10 +264,11 @@ def splice(self, wave):
         # Interpolate over gaps
         zeros = sensfunc_splice == 0.
         if np.any(zeros):
-            msgs.info("Interpolating over gaps")
+            msgs.info("Interpolating over gaps (and extrapolating with fill_value=1, if need be)")
             interp_func = scipy.interpolate.interp1d(wave_splice[np.invert(zeros)],
                                                  sensfunc_splice[np.invert(zeros)],
-                                                 kind='nearest', fill_value='extrapoloate')
+                                                 kind='nearest', fill_value=1., bounds_error=False) #
```

Comment:
Not sure whether we should fill 1.0, since the value of sensfunc could be much bigger than 1. Maybe fill with either 0 or nan and then set mask for those pixels?

Reply:
Replaced with 0